h/h_api/bulk_api/model/command.py

```diff
@@ -0,0 +1,133 @@
+from functools import lru_cache
+
+from h.h_api.bulk_api.model.config_body import Configuration
+from h.h_api.bulk_api.model.data_body import (
+    CreateGroupMembership,
+    UpsertGroup,
+    UpsertUser,
+)
+from h.h_api.enums import CommandType, DataType
+from h.h_api.model.base import Model
+from h.h_api.schema import Schema
+
+
+class Command(Model):
+    """A single abstract command provided to the API."""
+
+    validator = Schema.get_validator("bulk_api/wrapper.json")
+    validation_error_title = "Cannot interpret command as the wrapper is malformed"
+
+    def validate(self):
+        super().validate()
+        if isinstance(self.body, Model):
+            self.body.validate()
+
+    @classmethod
+    def create(cls, _type, body):
```

Comment:
```suggestion
    def create(cls, type_, body):
```

I think you're using the `_` to avoid a name conflict with the builtin `type`, rather than to indicate to PyLint that the argument is deliberately unused (which a leading underscore does). PEP 8 uses trailing underscores to avoid name conflicts

Reply:
Yep that's correct. I'll swap that over